src/test/java/com/worksap/nlp/sudachi/RegexOovProviderTest.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2022 Works Applications Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.worksap.nlp.sudachi
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class RegexOovProviderTest {
```

Comment:
Just to be sure, shouldn't we also test if the pattern matches non-single-byte characters? (To make sure the offset calculation in InputText is correct)

Reply:
Two tests use full-width characters which are non-single byte in utf-8